production/terraform/gcp/environments/terraform.tf

```diff
@@ -16,6 +16,11 @@ terraform {
   required_version = ">= 1.2.3"
 
   required_providers {
+    google = {
+      source = "hashicorp/google"
```

Comment:
could you align the `=` with the next line like this:
```
source  = "hashicorp/google"
version = "4.82.0"
```
otherwise it'd fail our `builders/tools/pre-commit` check. you could also run it which would fix issues like this for you.

Reply:
oh if you run builders/tools/pre-commit locally inside your repo, it should take care of the formatting. did it not?

Reply:
the one run by github is a different thing, quite experimental and yeah that one won't help fix it